keyboards/esca/getawayvan/keymaps/doof/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = STM32F042
+
+
+# Build Options
+#   comment out to disable the options.
+#
+
+
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no	# Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
+SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+CUSTOM_MATRIX = no # Custom matrix file
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = yes
+NO_USB_STARTUP_CHECK = no         	# Disable initialization only when usb is plugged in
+LTO_ENABLE = yes
```

Comment:
I wouldn't recommend using LTO here. 

If you need space, there are a couple of other ways to go about this.